physics/cires_ugwp_initialize_v1.F90

```diff
@@ -0,0 +1,799 @@
+!===============================
+! cu-cires ugwp-scheme
+!    initialization of selected 
+!  init gw-solvers (1,2,3,4)
+!  init gw-source specifications
+!  init gw-background dissipation
+!==============================
+!
+! Part-0   specifications of common constants, limiters and "criiical" values
+! 
+!
+    
+    module ugwp_common_v1
+!
+!     use machine,  only : kind_phys
+!     use physcons, only : pi => con_pi, grav => con_g, rd => con_rd,   &
+!                          rv => con_rv, cpd => con_cp, fv => con_fvirt,&
+!                          arad => con_rerth
+     implicit none
+
+      real, parameter ::  grav =9.81, cpd = 1004.
```

Comment:
Leaving as-is for now, but am eliminating "use" statements which call for the physical constants.

Reply:
IMO, it is dangerous for every scheme to be defining their own physical constants. They should be defined in one place and used across physics. In the CCPP, sending constants through the argument list enables this -- each host model that uses the CCPP defines physical constants in their own way (which is why you can't rely on "use physcons") and as long as the host provides metadata for the constants so that the CCPP knows what to pass in, it all works as expected with common values amongst physics. In practice, for FV3, the constants may be defined in physcons.F90, but they have metadata and this metadata is made known to the CCPP framework at build time.

Reply:
I am still seeing places in the code that do import constants from `module ugwp_common_v1`? Did I misread your reply?

Reply:
I see what you mean, but for the subroutines that are actually called in the ugwp_v1 version, the physical constants that are used are passed in through the argument lists.  I would like to leave the other subroutines as-is, as I believe a lot of them will be removed after cleaning up (with Valery next month).